src/components/Trips.js

```diff
@@ -12,6 +16,12 @@ function Trips() {
     setTrips(trip);
   }, []);
 
+  function handleRemoveTrip(trip) {
+    removeTripFromLocalStorage(trip);
+    const newTrips = getTripsFromLocalStorage();
+    setTrips(newTrips);
+  }
+
   function renderTrips() {
     return trips.map((trips, index) => {
       return (
```

Comment:
What you are returning in here could be a component as well, (`TripCard` or something like this)